integration-cli/docker_cli_build_test.go

```diff
@@ -6111,6 +6112,7 @@ func (s *DockerSuite) TestBuildBuildTimeArgEnv(c *check.C) {
 		RUN [ "$(env | grep FOO7)" == "" ]
 		RUN [ "$(env | grep FOO8)" == "" ]
 		RUN [ "$(env | grep FOO9)" == "" ]
+		RUN [ "FO10" == "" ]
```

Comment:
Looks like one line got lost in the cherry-pick and has to go here; https://github.com/mlaventure/docker/blob/10119047e6769ac17f8051fcd90ae5a4c761ae1e/integration-cli/docker_cli_build_test.go#L6092

<img width="793" alt="screen shot 2017-05-19 at 02 54 28" src="https://cloud.githubusercontent.com/assets/1804568/26229012/b955ffb0-3c3e-11e7-8e07-c0162f062b3c.png">